_posts/2021-05-23-irc-move.md

```diff
@@ -0,0 +1,41 @@
+
+## What has happened?
+
+This will be a very short explanation again. This time, we will condense what apparently has happened behind the scenes of freenode. If you want to paint your own picture, read [this post](https://gist.github.com/realrasengan/88549ec34ee32d01629354e4075d2d48) from one side of this drama, and [this resigning letter](https://gist.github.com/aaronmdjones/1a9a93ded5b7d162c3f58bdd66b8f491) from the other. There are many more resigning letters around, you can search for it.
+
+In a nutshell, about a dozen volunteer developers behind freenode left the project. This happened because Andrew Lee bought freenode Limited in 2017, a company which was used as a legal representation of freenode, mostly for their freenode Live Conferences. While he promised to not obtain any influence over freenode itself, this apparently changed this week. Andrew and his team gained full controll over the network (as stated by their [twitter account](https://twitter.com/freenodestaff/status/1395046345145307140)), which lead to the volunteers resign. Whilste resigning, mosted of the volunteers stated they will now be at Libera.Chat.
+
+## Why is NewPipe moving?
+
+Honestly, we do not trust an American business man saying he won't do anything with our data. We do trust developers, we are developers ourself. When you do something to a project which leads to its core team resigning, and said core team starts a new project, we will join this new project. We like our IRC channels, they are the best way to communicate with each other while staying True to our privacy values. After discussing the events internally, we decide to switch the network, and we are in good company doing so. 
+
+## What do I have to do?
+
+You won't have to do much if you use an IRC client. Just point it to Libera.Chat with their link, [irc.libera.chat:6697](ircs://irc.libera.chat:6697), and you will find all the familiar faces in `#newpipe`. You can find more connection help on their [help site](https://libera.chat/guides/connect).
+
+If you use Matrix, you will need to wait a bit until the Matrix team finalizes their bridge. NewPipe will continue support for Matrix though, and we will only fully close the freenode channel once Matrix is up and running.
+
+## But I expected a new NewPipe version :(
+
+No worries, we will release a new version soon. And then link to it right here. It will be a calm release with a nice new feature, just the right thing after reading through this drama.
```

Comment:
```suggestion
No worries, we will release a new version soon. And then link to it right here. It will be a calm release with a nice new feature, just the thing we all need after reading through this drama.
```